src/includes/entry.h

```diff
@@ -0,0 +1,36 @@
+#ifndef UI_NODE_ENTRY
+#define UI_NODE_ENTRY 1
+
+#define DEFINE_ENTRY_METHODS()                                                 \
```

Comment:
DEFINE_ENTRY_METHODS, INHERITS_ENTRY_METHODS and DECLARE_ENTRY_METHODS are not needed anymore, since nbind correctly handles class inheritance by now.